### PR TITLE
Fix the vision UI, now use CompressImage

### PR DESCRIPTION
**Pull request by marcantoinecouture**
## Description
The VisionUi tool now use the CompressImage

## Checklist
- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas (NA)
- [X] I have made corresponding changes to the documentation (NA)
- [X] My changes generate no new warnings

### CODE REVIEW

File: rqt_vision/src/rqt_vision/VisionMainWidget.py [hunk 1]
```diff
@@ -114,17 +114,21 @@ def current_execution_index_changed(self, index):
             self._filterchain = self._srv_get_filterchain_from_execution(self._current_execution)
         except rospy.ServiceException as err:
             rospy.logerr(err)
+        
+        topic_name = '/proc_image_processing/' + new_execution + '_image/compressed'
```
Comment (kevincharbonneau) [inline rqt_vision/src/rqt_vision/VisionMainWidget.py]: you could use `format()` to insert string into line like so : 
```python
topic_name = "/proc_image_processing/{}_image/compressed".format('new_execution')
```

**Review by kevincharbonneau — CHANGES_REQUESTED** [review]
I would recommend using `format()`.

File: rqt_vision/src/rqt_vision/VisionMainWidget.py [hunk 2]
```diff
@@ -114,17 +114,21 @@ def current_execution_index_changed(self, index):
...
+        
         try:
             self._srv_start_republisher('/proc_image_processing/' + new_execution + '_image', 1)
             ip = os.getenv('ROS_IP', '127.0.0.1')
             topic_name = '/proc_image_processing/' + new_execution + '_image_' + ip.replace('.', '')
```
Comment (gauthiermartin) [inline rqt_vision/src/rqt_vision/VisionMainWidget.py]: Use "".format() here

**Review by gauthiermartin — CHANGES_REQUESTED** [review]
Please refactor you code to use the best practices 👍

**Comment by marcantoinecouture** [comment]
Thank you guys for the comment

**Review by kevincharbonneau — APPROVED** [review]
LGTM ! 👍

**Pull request merged**